azure-pipelines-templates/check-mdp-for-build.yml

```
@@ -0,0 +1,93 @@
+# Copyright (c) .NET Foundation and Contributors
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)